infra/aws/terraform/management-account/accounts.tf

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_organizations_account" "organisation_security" {
+  name                       = "k8s-infra-organisation-security"
+  email                      = "k8s-infra-aws-admins+security@kubernetes.io"
+  iam_user_access_to_billing = "ALLOW"
+  parent_id                  = aws_organizations_organizational_unit.security.id
+
```

Comment:
Are we happy with the default `role_name`? I prefer to override it (and also, I often only use the auto-vended role for bootstrapping)

Reply:
Are you suggesting a custom role or a predefined role ?

Reply:
By default, AWS names the new role `OrganizationAccountAccessRole`.

https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html

I prefer to make a new role with a different name such as `TerraformBootstrap` or something like that. _However_, if we have already started having an assumption that the role name is `OrganizationAccountAccessRole`, maybe we should stick with it.

Reply:
Defaulted to the role to `OrganizationAccountAccessRole`.